introduction.md

```diff
@@ -0,0 +1,9 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent fermentum scelerisque nunc. Vestibulum et tincidunt felis, et pellentesque leo. Morbi ut feugiat metus. Nulla vel odio lorem. Curabitur placerat risus sit amet eleifend tempus. Suspendisse libero libero, elementum id felis a, congue convallis magna. Sed eget rutrum neque.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.